core/src/main/java/io/th0rgal/oraxen/api/OraxenBlocks.java

```diff
@@ -253,16 +259,19 @@ private static void removeNoteBlock(Block block, @Nullable Player player, boolea
             if (forceDrop || player.getGameMode() != GameMode.CREATIVE)
                 drop = noteBlockBreakEvent.getDrop();
 
-            block.getWorld().sendGameEvent(null, GameEvent.BLOCK_DESTROY, block.getLocation().toVector());
+            World world = block.getWorld();
+
+            world.sendGameEvent(player, GameEvent.BLOCK_DESTROY, loc.toVector());
```

Comment:
I avoided player here as it caused some issues in testing with crashes
Not sure the player matters much here but

Reply:
if a sculk sensor gets activated it won't power any sculk shriekers because it wasn't from a player